swan-lake/development-tutorials/test-document-the-code/test-ballerina-code/configure-tests.md

```diff
@@ -32,48 +33,49 @@ function beforeFunc() {
 }
 
 // Test function.
-@test:Config {}
+@test:Config
 function testFunction1() {
     io:println("I'm in test function 1!");
     test:assertTrue(true, msg = "Failed");
 }
 
 // Test function.
-@test:Config {}
+@test:Config
 function testFunction2() {
     io:println("I'm in test function 2!");
     test:assertTrue(true, msg = "Failed");
 }
 ```
 
-#### The `@test:AfterSuite {}` annotation
-The `AfterSuite` annotated function will be run once after all the tests in the test suite are run. This can be used for 
-cleaning up the test suite level aspects. A test suite covers tests related to a module.
+#### The `@test:AfterSuite` annotation
+
+The `AfterSuite` annotated function will be run once after all the tests in the test suite are run. This can be used for test suite level clean up. A test suite covers tests related to a module.
```

Comment:
```suggestion
The `AfterSuite` annotated function will be run once after all the tests in the test suite are run. This can be used for cleaning up at the test suite level. A test suite covers tests related to a module.
```